hummingbot/client/command/start_command.py

```diff
@@ -76,6 +78,25 @@ async def start_check(self,  # type: HummingbotApplication
         self._notify(f"\nStatus check complete. Starting '{self.strategy_name}' strategy...")
         if global_config_map.get("paper_trade_enabled").value:
             self._notify("\nPaper Trading ON: All orders are simulated, and no real orders are placed.")
+
+        # Show warning message if the exchange connector has outstanding issues or not working
+        connector = get_strategy_config_map(self.strategy_name).get("exchange").value
+        status = get_connector_status(get_strategy_config_map(self.strategy_name).get("exchange").value)
+
+        # Display custom message for connectors on RED or YELLOW status
+        if connector == "okex":
```

Comment:
A better way of doing this is to create another dictionary in `connector_status.py`, and retrieve and display messages here.
For example:
```python
warning_msg = get_connector_warning(connector)
if warning_msg is not None:
    self._notify(....
```

Reply:
Thanks @Nullably! I just pushed a new commit with these changes.